desktop-web-app/web-app-vue/rpc/prefs-client.ts

```diff
@@ -0,0 +1,44 @@
+/*
+ *  Copyright (c) 2025 TeamDev
+ *
+ *  Permission is hereby granted, free of charge, to any person obtaining a copy
+ *  of this software and associated documentation files (the "Software"), to deal
+ *  in the Software without restriction, including without limitation the rights
+ *  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ *  copies of the Software, and to permit persons to whom the Software is
+ *  furnished to do so, subject to the following conditions:
+ *
+ *  The above copyright notice and this permission notice shall be included in all
+ *  copies or substantial portions of the Software.
+ *
+ *  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ *  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ *  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ *  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ *  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ *  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ *  SOFTWARE.
+ */
+
+import {createGrpcWebTransport} from "@connectrpc/connect-web";
+import {createClient} from "@connectrpc/connect";
+import {PrefsService} from "@/gen/prefs_pb.ts";
+
+/**
+ * A port for RPC communication obtained from the server side via
+ * the JxBrowser's Java-Js bridge.
+ */
+declare const rpcPort: number;
```

Comment:
Is there a way, we can re-use all non-UI specific TypeScript too? @ivan-diachenko-td @andrii-kulminskyi is it possible without overcomplicating the build?